v19.2/table-expressions.md

```diff
@@ -326,15 +326,23 @@ For example:
 Syntax:
 
 ~~~
-[ <statement> ]
+SELECT .. FROM [ <stmt> ]
 ~~~
 
-An [explainable statement](sql-grammar.html#preparable_stmt)
-between square brackets in a table expression context designates the
-output of executing said statement.
+A [statement](sql-grammar.html#row_source_extension_stmt) between square brackets in a table expression context designates the output of executing the statement as a row source. The following statements are supported as row sources for table expressions:
+
+- [`DELETE`](delete.html)
+- [`EXPLAIN`](explain.html)
+- [`INSERT`](insert.html)
+- [`SELECT`](select.html)
+- [`SHOW`](sql-statements.html#data-definition-statements)
+- [`UPDATE`](update.html)
+- [`UPSERT`](upsert.html)
+
+ `SELECT .. FROM [ <stmt> ]` is equivalent to `WITH table_expr AS ( <stmt> ) SELECT .. FROM table_expr`
 
 {{site.data.alerts.callout_info}}
-This is a CockroachDB extension. This syntax complements the [subquery syntax using parentheses](#subqueries-as-table-expressions), which is restricted to [selection queries](selection-queries.html). It was introduced to enable use of any [explainable statement](sql-grammar.html#preparable_stmt) as subquery, including `SHOW` and other non-query statements.
+This CockroachDB extension syntax complements the [subquery syntax using parentheses](#subqueries-as-table-expressions), which is restricted to [selection queries](selection-queries.html). It was introduced to enable the use of [statements](sql-grammar.html#row_source_extension_stmt) as subqueries.
```

Comment:
This mention of subqueries could be a link to https://www.cockroachlabs.com/docs/v19.2/subqueries.html.